android/app/build.gradle

```diff
@@ -206,18 +196,6 @@ android {
         }
     }
 
-    flavorDimensions "version"
-    productFlavors {
-        gps {
-            applicationId 'org.pathcheck.covidsafepaths'
-            dimension "version"
-        }
-        bt {
-            applicationId 'org.pathcheck.covidsafepathsBt'
-            dimension "version"
-        }
-    }
```

Comment:
We no longer need product flavors.
The new build variants are `debug`, `staging`, `release`